### PR TITLE
documented packet 52 and 40

### PR DESCRIPTION
**Pull request by unknown81311**

### CODE REVIEW

File: docs/networking/packets/40-glowstick-effect-removal.mdx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+# Glowstick Effect Removal
+
+This packet is sent when the player removes a glowstick effect.
+
+- **ID**: 0x28 (40)
+- **Size**: 6
+- **LZ4 Compressed**: No
+- **State**: Playing
+- **Bound To**: Client IDK Server
```
Comment (TechnologicNick) [inline docs/networking/packets/40-glowstick-effect-removal.mdx]: Please add the direction.

File: docs/networking/packets/40-glowstick-effect-removal.mdx [hunk 2]
```diff
@@ -0,0 +1,23 @@
+---
+title: 40 - Glowstick Effect Removal WIP
+---
+
+# Glowstick Effect Removal
+
+This packet is sent when the player removes a glowstick effect.
```
Comment (TechnologicNick) [inline docs/networking/packets/40-glowstick-effect-removal.mdx]: I doubt the devs made a packet specifically for the glowstick effect removal.

File: docs/networking/packets/40-glowstick-effect-removal.mdx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- **State**: Playing
+- **Bound To**: Client IDK Server
+
+## Structure
+
+| Offset | Field Name       | Field Type | Notes |
+|--------|------------------|------------|-------|
+| 0x00   | Unknown          | u8         | Unknown |
+| 0x01   | Unknown          | u8         | Unknown |
+| 0x02   | Effect ID        | u32        |  |
```
Comment (TechnologicNick) [inline docs/networking/packets/40-glowstick-effect-removal.mdx]: Add endianness maybe?

File: docs/networking/packets/52-lift-level.mdx [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+# Lift Level
+
+This packet is sent when the player presses up or down, lift has to be placed.
+
+- **ID**: 0x34 (52)
+- **Size**: 10
+- **LZ4 Compressed**: No
+- **State**: Playing
+- **Bound To**: Client IDK Server
```
Comment (TechnologicNick) [inline docs/networking/packets/52-lift-level.mdx]: Direction pls

**Review by TechnologicNick — CHANGES_REQUESTED** [review]

**Review by ColdMeekly — APPROVED** [review]
nice

**Review by ColdMeekly — APPROVED** [review]

**Review by TechnologicNick — APPROVED** [review]

**Pull request merged**